acsoo/templates/project/+project.name+/setup.py.bob

```diff
@@ -14,5 +14,9 @@ setup(
     install_requires=[
         'click-odoo-contrib',
     ],
-    odoo_addons=True,
+    odoo_addons={
+        'depends_override': {
+            'module_auto_update': 'odoo{{{ odoo.major }}}-addon-module_auto_update>={{{ odoo.major }}}.0.2.0.0',
```

Comment:
@sbidoul When I first read the code, I wanted to propose this addition however I'm not sure how useful this last commit is... It is indeed based on the principle that versions are kept across the different Odoo series. But we do not have our hands on this convention...

Reply:
In general you are right. In this specific case it will be ok for 8 to 11. And we must really avoid other versions :)